openfisca_us/parameters/irs/payroll/medicare/additional/exclusion.yaml

```diff
@@ -0,0 +1,22 @@
+description: Earnings exclusion above which the Additional Medicare Tax is levied (same for wages and self-employment earnings)
```

Comment:
I would favour separating these, but this works for this PR, thanks.

Reply:
Filed #664